lib/api/controllers/serverController.js

```diff
@@ -315,6 +318,34 @@ class ServerController extends NativeController {
       throw kerror.get('api', 'assert', 'unexpected_argument', scope, ['kuzzle', 'app', 'all']);
     }
 
+    if (exclude.length) {
+      definition.tags = definition.tags.filter(tag => ! exclude.includes(tag.name));
+      Object.entries(definition.paths).forEach(([path, pathDefinition]) => {
```

Comment:
You should use `for ... of ` loop instead of `Array.forEach`.
`Array.forEach` does not comply with `await` keyword and can leads to unwanted behavior so we stick to the ` for ... of ` loop only